cmd/terramate/cli/cli.go

```diff
@@ -2608,7 +2609,7 @@ func (c *cli) cfg() *config.Root            { return c.prj.root }
 func (c *cli) baseRef() string              { return c.prj.baseRef }
 func (c *cli) stackManager() *stack.Manager { return c.prj.stackManager }
 func (c *cli) rootNode() hcl.Config         { return c.prj.root.Tree().Node }
-func (c *cli) cred() credential             { return c.cloud.client.Credential.(credential) }
+func (c *cli) cred() auth.Credencial        { return c.cloud.client.Credential.(auth.Credencial) }
```

Comment:
This is a typo in English
```suggestion
func (c *cli) cred() auth.Credential        { return c.cloud.client.Credential.(auth.Credential) }
```

Reply:
oh indeed, It's mistype.